src/api/.rubocop.yml

```diff
@@ -224,3 +224,7 @@ Style/MethodCallWithArgsParentheses:
     - write
     - yield
     - ymp
+
+Layout/AlignHash:
+  Enabled: true
+
```

Comment:
Nitpick: as the cop is enabled by default this lines are not needed.

Reply:
As @eduardoj says this is enabled by default. But removing it is IMO not a nitpick. We have already agreed to follow the defaults for this cop, which is not what is done if you add it here.